lms/djangoapps/courseware/model_data.py

```diff
@@ -1024,4 +1025,4 @@ def get_score(user_id, usage_key):
     except StudentModule.DoesNotExist:
         return None
     else:
-        return student_module.grade, student_module.max_grade
+        return student_module
```

Comment:
It contains all of these things; most of which are, strictly speaking, a superset of the data specifically related to "scores". However, `set_score` above is still directly calling `StudentModule.objects.get_or_create()`, so the inconsistency in referring to a StudentModule object as a "score" is consistent between the two methods.

```
(Pdb) dir(student_module)
['DONE_TYPES', 'DoesNotExist', 'MODEL_TAGS', 'MODULE_TYPES', 'MultipleObjectsReturned', '__class__', '__delattr__', '__dict__', '__doc__', '__eq__', '__format__', '__getattribute__', '__hash__', '__init__', u'__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__setstate__', '__sizeof__', '__str__', '__subclasshook__', '__unicode__', '__weakref__', '_base_manager', '_check_column_name_clashes', '_check_field_name_clashes', '_check_fields', '_check_id_field', '_check_index_together', '_check_local_fields', '_check_long_column_names', '_check_m2m_through_same_relationship', '_check_managers', '_check_model', '_check_ordering', '_check_swappable', '_check_unique_together', '_default_manager', '_deferred', '_do_insert', '_do_update', '_get_FIELD_display', '_get_next_or_previous_by_FIELD', '_get_next_or_previous_in_order', '_get_pk_val', '_get_unique_checks', '_meta', '_perform_date_checks', '_perform_unique_checks', '_save_parents', '_save_table', '_set_pk_val', '_state', 'all_submitted_problems_read_only', 'check', 'clean', 'clean_fields', 'course_id', 'created', 'date_error_message', 'delete', 'done', 'from_db', 'full_clean', 'get_deferred_fields', 'get_done_display', 'get_module_type_display', 'get_next_by_created', 'get_next_by_modified', 'get_previous_by_created', 'get_previous_by_modified', 'grade', 'id', 'max_grade', 'modified', 'module_state_key', 'module_type', 'objects', 'pk', 'prepare_database_save', 'refresh_from_db', 'save', 'save_base', 'serializable_value', 'state', 'student', 'student_id', 'studentmodulehistory_set', 'studentmodulehistoryextended_set', 'unique_error_message', 'validate_unique']
```